### PR TITLE
add template for sensu-cluster-metrics repo

### LINKED ISSUE #11: add sensu-cluster-metrics template

### PR DESCRIPTION
**Pull request by jspaleta**
Closes #11

### CODE REVIEW

File: monitors/sensu/metrics-sensu-cluster.yaml [hunk 1]
```diff
@@ -0,0 +1,119 @@
+---
+# Description
+#
+#   Generate cluster metrics from Sensu using the Sensu graphql endpoint
+#
+# Instructions
+#   1. Ensure there is an Sensu api-key for a user with appropriate cluster-wide read access
```
Comment (nixwiz) [inline monitors/sensu/metrics-sensu-cluster.yaml:7]: Should this be "a Sensu api-key"?

File: monitors/sensu/metrics-sensu-cluster.yaml [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+    url: https://assets.bonsai.sensu.io/39369b9db9846b0a437bcff1be0f47ab9012a31e/sensu-cluster-metrics_0.0.1_linux_386.tar.gz
+  - filters:
+    - entity.system.os == 'linux'
+    - entity.system.arch == 'amd64'
+    headers: null
+    sha512: efc446cfb9920f151409a45fe1297bf0bd88871944585516d7700303c81a4c642cf0ea31d38d9fc3d8e42677cbc0fc4f4a8d3fea0f670afb2de9362943ae8dfb
+    url: https://assets.bonsai.sensu.io/39369b9db9846b0a437bcff1be0f47ab9012a31e/sensu-cluster-metrics_0.0.1_linux_amd64.tar.gz
+  filters: null
+  headers: null
+
```
Comment (nixwiz) [inline monitors/sensu/metrics-sensu-cluster.yaml:117]: Kill the extraneous blank lines at the bottom

**Review by nixwiz — COMMENTED** [review]
Just a couple of cleanup items.

File: monitors/sensu/metrics-sensu-cluster.yaml (hunk 1 above)
Comment (jspaleta) [reply monitors/sensu/metrics-sensu-cluster.yaml]: should be fixed in latest commit

File: monitors/sensu/metrics-sensu-cluster.yaml (hunk 2 above)
Comment (jspaleta) [reply monitors/sensu/metrics-sensu-cluster.yaml]: should be fixed in latest commit

**Review by nixwiz — APPROVED** [review]
LGTM 👍

**Pull request merged**